app/src/main/java/org/simple/clinic/editpatient/PatientData.kt

```diff
@@ -0,0 +1,59 @@
+package org.simple.clinic.editpatient
+
+import org.simple.clinic.editpatient.PatientEditValidationError.COLONY_OR_VILLAGE_EMPTY
+import org.simple.clinic.editpatient.PatientEditValidationError.DISTRICT_EMPTY
+import org.simple.clinic.editpatient.PatientEditValidationError.FULL_NAME_EMPTY
+import org.simple.clinic.editpatient.PatientEditValidationError.PHONE_NUMBER_EMPTY
+import org.simple.clinic.editpatient.PatientEditValidationError.PHONE_NUMBER_LENGTH_TOO_LONG
+import org.simple.clinic.editpatient.PatientEditValidationError.PHONE_NUMBER_LENGTH_TOO_SHORT
+import org.simple.clinic.editpatient.PatientEditValidationError.STATE_EMPTY
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.PatientPhoneNumber
+import org.simple.clinic.registration.phone.PhoneNumberValidator
+import org.simple.clinic.registration.phone.PhoneNumberValidator.Result
+import org.simple.clinic.registration.phone.PhoneNumberValidator.Type
+
+data class PatientData(
+    val name: String,
+    val gender: Gender,
+    val phoneNumber: String,
+    val colonyOrVillage: String,
+    val district: String,
+    val state: String
+) {
```

Comment:
This name is not very convincing with this class name. We're going to end up with `Patient`, `PatientProfile`, `OngoingPatientEntry` and `PatientData`. 

What do you think about splitting `OngoingPatientEntry` into `OngoingNewPatientEntry` and `OngoingEditPatientEntry` or something?

Reply:
I like this. I'll make this change.